### DISCUSSION TITLE
Migration from Plex to Jellyfin - Media Naming

### DISCUSSION THREAD

**Discussion by Bammede** [jellyfin/jellyfin#6111] [Q&A]
I use Plex so my media directory naming is like this:

1. for movies: Movies > Movie_Tittle (year) > Movie_Tittle (year) - 1080p.mkv, Movie_Tittle (year) - 2160p.mkv
2. for tv shows: TV Shows > Show_Tittle (year) > Season 01 > Show_Title - s01e01.mkv ...
3. for music: Music > Artist_Name - Album_Title > songtitle.flac ...
4.  for audiobooks: Audiobooks > Author_Name - Book_Title > title1.mp3, title2.mp3 ...

Will jellyfin grab metadata correctly with this naming?

**Comment by crobibero** [comment]
Yes, that should be matched.
Our naming docs: https://jellyfin.org/docs/general/server/media/movies.html
